webapp/packages/plugin-sql-editor-navigation-tab/src/SqlEditorBootstrap.ts

```diff
@@ -169,6 +182,44 @@ export class SqlEditorBootstrap extends Bootstrap {
       },
     });
 
+    this.menuService.addCreator({
+      isApplicable: context => {
+        const tab = context.tryGet(DATA_CONTEXT_SQL_EDITOR_RESULT_ID);
+        const state = context.tryGet(DATA_CONTEXT_TABS_CONTEXT);
+        return !!tab && !!state?.enabledBaseActions && context.get(DATA_CONTEXT_MENU) === MENU_TAB;
```

Comment:
you can use `menu` field of creator instead of `context.get(DATA_CONTEXT_MENU) === MENU_TAB`